python-interpreter-builder/Dockerfile.in

```diff
@@ -29,7 +29,6 @@ RUN apt-get update && apt-get install -yq \
     net-tools \
     netbase \
     python3 \
-    quilt \
```

Comment:
I removed the last patch that uses `quilt`.  We'll add it back in if we add another local patch.

Actually, `quilt` is pretty annoying, in that it has a fatal error if you give it a list of patches that happens to be empty.  Otherwise I would have left all the quilt code intact.